homeassistant/components/deconz/__init__.py

```diff
@@ -143,7 +144,17 @@ def async_configure(call):
     hass.services.async_register(
         DOMAIN, 'configure', async_configure, schema=SERVICE_SCHEMA)
 
-    hass.bus.async_listen_once(EVENT_HOMEASSISTANT_STOP, deconz.close)
+    @callback
+    def deconz_shutdown(event):
+        """
+        Wraps the call to deconz.close. Used as an argument to
```

Comment:
The docstring needs to begin with a one line header sentence. Then a blank line, if more sentences follow.

Reply:
Also start the docstring in imperative mood.

Reply:
Oops, sorry. I haven't contributed in a while, forgot the guidelines.